debian/changelog

```diff
@@ -1,3 +1,54 @@
+ubuntu-advantage-tools (27.1) impish; urgency=medium
+
+  * d/control:
+    - specify debianutils min version
+  * d/changelog:
+    - fix lintian typos amend and redact incorrect 27.0 entry (GH: #1624)
+  * lintian:
+    - override ubuntu-advantage-pro wanted-by-target cloud-init
+    - override xenial specific errors
+    - rename package-specific overrides for pro vs tools
+  * New upstream release 27.1:
+    - apt-hook:
+      + avoid segfault when comparing null Apt file origin to esm
+        (LP: #1929123)
+      + avoid wrapping static message formats at 80 chars
+      + update go build flags based on lintian warnings (GH: #1626)
+      + only add newlines for MOTD if message file length is non-zero
+    - attach: do not print contract name if empty
+    - autocomplete: Do not show beta services in autocomplete (GH: #1594)
+    - cis:
+      + make service non-beta
+      + post enable message pointing to docs
+      + update cis help url
+    - docs: update releases.md per SRU review feedback on branch structuring
+    - enable: correct messaging for beta service (GH: #1588)
+    - errors: print a more helpful message when ssl fails (GH: #1618)
+    - fips:
+      + Block enabling fips if fips-updates once enabled (GH: #1600)
+      + clarify messaging (GH: #1631)
```

Comment:
I think we can update this message with the issue title:
`Update output of fips commands`

Just so it will be more clearer